bot/utils/helpers.py

```diff
@@ -41,3 +47,19 @@ def remove_subdomain_from_url(url: str) -> str:
     netloc = extracted_url.registered_domain
     parsed_url = parsed_url._replace(netloc=netloc)
     return parsed_url.geturl()
+
+
+async def try_handle_forbidden(error: Forbidden, message: Message | None = None) -> bool:
```

Comment:
This function name, and docstring, should be updated to better reflect what it's doing. Something like `handle_forbidden_from_block`.

Reply:
If we ever find the need to do that then we can come up with a better name at that time, but for now it's needlessly mysterious to the caller as to what it does.

Reply:
Fair enough

Reply:
e4b2d5e